config/environments/test.rb

```diff
@@ -18,8 +18,6 @@
   # recommended that you enable it in continuous integration systems to ensure eager
   # loading is working properly before deploying your code.
   config.eager_load = ENV["CI"].present?
-  # cache classes on CI, but enable reloading for local work (bin/rspec)
-  config.enable_reloading = ENV["CI"].blank?
 
```

Comment:
```suggestion

# cache classes on CI, but enable reloading for local work (bin/rspec)
  config.enable_reloading = ENV["CI"].blank?
```

I mad this change recently, apparently missed the other setting above while doing so. Can you  revert the changes in this file or put this back in? What was the issue here?

Reply:
Did not mean to commit this. I enable reloading locally so i can run tests.